Library/Formula/pkcrack.rb

```diff
@@ -0,0 +1,185 @@
+class Pkcrack < Formula
+  homepage "https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack.html"
+  url "https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack/pkcrack-1.2.2.tar.gz"
+  version "1.2.2"
+  sha256 "4d2dc193ffa4342ac2ed3a6311fdf770ae6a0771226b3ef453dca8d03e43895a"
+
+  # Apply to the patch to build on the OSX
+  patch :DATA
+  def install
+    system "make", "-C", "src/"
+
+    # ln -s src/[BINARY] to ${prefix}/bin
+    bin.install "src/pkcrack"
```

Comment:
Is there anything other than executables in the ` src ` directory? Otherwise ` bin.install Dir["src/*"] ` works. Otherwise: ` bin.install Dir["src/*"].select {|f| File.executable? f} ` would look nicer.